tests/Feature/Connect/BootstrapsConnect.php

```diff
@@ -15,6 +15,8 @@ protected function setUp(): void
     {
         parent::setUp();
 
+        // config(['feature.aggregate_queries' => true]);
```

Comment:
Is this still needed/a reference to a TODO?

Reply:
It's a way to toggle the tests to run against the aggregate queries code, which was a path not being tested, letting this problem through. I don't know if there's an easy way to test both code paths, so I left this in as an easy toggle.